app/controllers/users_controller.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+class UsersController < ApplicationController
+  def index
+    @users = User.all
```

Comment:
btw do we need this feature, like listing all user in the current system 🤔

Reply:
I want a page to redirect after signed up and show that username that user just signed up is stored to db.
In this case, is it enough to just investigate database?

Reply:
Well in the real project we don't do that :D think about the feature as a user :) I think in this case, for UX we don't need to show anything to proof the user is created, moving on we will redirect to the Sign In page with a Flash message like "Your account is created successfully"....

And to make sure the Account is created, I saw your test it in the Controller test already, so that can be use for proof of work, making sure the user is created.

What do you think?

Reply:
```
moving on we will redirect to the Sign In page with a Flash message like "Your account is created successfully"
```
Do I need to create Sign in page with showing flash message in this PR?

Reply:
No need for now @gutakk step by step, for now just redirect to the `root page` with the flash message is enough

Reply:
Solved